prov/efa/src/rdm/rxr_rma.c

```diff
@@ -412,6 +464,7 @@ ssize_t rxr_rma_writemsg(struct fid_ep *ep,
 		err = -FI_EAGAIN;
 		goto out;
 	}
+	rxr_op_entry_prepare_to_post_write(tx_entry);
```

Comment:
I think this is used only in true write.

Reply:
It is needed in both rxr_rma_post_shm_write (where it was already being called... ) and in `rxr_op_entry_post_remote_write`.

I removed the call from `rxr_rma_writemsg` and included it in `rxr_op_entry_post_remote_write`.